### PR TITLE
Jc/alert boxes

### PR DESCRIPTION
**Pull request by jarsbe**
Added a start script to run gulp and the serve to resolve https://www.pivotaltracker.com/n/projects/1187904/stories/92808932 and fixed the text on a danger alert to resolve https://www.pivotaltracker.com/n/projects/1187904/stories/92804110.

screen shot 2015-04-20 at 11 06 03

### CODE REVIEW

**Comment by pmvieira** [comment]
welcome jack!
Have a merry first PR! :notes:

**Comment by jarsbe** [comment]
@Dakuan Ready to go

File: build/qboot.css [hunk 1]
```diff
@@ -6707,6 +6707,9 @@ i {
 .btn-row .btn:last-child {
   margin-right: 0;
 }
+.alert-danger {
+  color: white;
```
Comment (Dakuan) [inline build/qboot.css]: awesome, works really well!

I wonder if we can use a bootstrap variable instead

**Comment by Dakuan** [comment]
@jarsbe give that bootstrap variable a go then :shipit:

**Pull request merged**